cmp/grid/impl/ColumnHeader.js

```diff
@@ -217,7 +217,10 @@ class LocalModel {
     }
 
     autosize() {
-        this.gridModel?.autosizeAsync({columns: this.colId, showMask: false});
+        const {gridModel} = this;
+        if (gridModel?.autosizeEnabled) {
```

Comment:
Do we need this check, since `autosizeAsync` also checks?

Reply:
True, but would like to eventually come back here and change it so that it actually "consumes" the event and calls the method.   

Generally, see the no-op in the method more as an extra check